java/ql/lib/config/semmlecode.dbscheme

```diff
@@ -588,6 +588,14 @@ imports(
   unique int id: @import,
   int holder: @classorinterfaceorpackage ref,
   string name: string ref,
+  /* kind:
+      1 = TYPE
+      2 = TYPEWILDCARD
+      3 = PACKAGEWILDCARD
+      4 = TYPEWILDCARDSTATIC
+      5 = TYPEMEMBERSTATIC
+      6 = MODULEIMPORT
+    */
```

Comment:
You seem to be forcing a dbscheme change here by adding a comment. It seems we can just completely omit a scheme change here (and not have upgrade and downgrade scripts). Older versions of CodeQL not knowing how to interpret 6 should not be a problem.